data/src/main/java/com/eleks/data/di/FirebaseModuleBinder.kt

```diff
@@ -2,18 +2,39 @@ package com.eleks.data.di
 
 import com.eleks.data.firebase.source.FirebaseDataSource
 import com.eleks.data.firebase.source.impl.FirebaseDataSourceImpl
-import dagger.Binds
+import com.google.firebase.auth.FirebaseAuth
+import com.google.firebase.firestore.FirebaseFirestore
+import com.google.firebase.firestore.ktx.firestore
+import com.google.firebase.ktx.Firebase
 import dagger.Module
+import dagger.Provides
 import dagger.hilt.InstallIn
 import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
 
 @Module
 @InstallIn(SingletonComponent::class)
-abstract class FirebaseModuleBinder {
+object FirebaseModuleBinder {
 
-    @Binds
-    abstract fun bindFirebaseDataSource(
-        firebaseDataSourceImpl: FirebaseDataSourceImpl
-    ): FirebaseDataSource
+    @Singleton
+    @Provides
+    fun provideFirestoreFirebase(): FirebaseFirestore {
+        return Firebase.firestore
+    }
+
+    @Singleton
+    @Provides
+    fun provideFirebaseAuth(): FirebaseAuth {
+        return FirebaseAuth.getInstance()
+    }
+
+    @Singleton
+    @Provides
+    fun bindFirebaseDataSource(
```

Comment:
I suppose we don't need this provider. Just use bind. Take a look on RepositoryModuleBinder